utils/smartpaste/githubcode.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Rule} from 'turndown';
+
+export function githubCodeTurndownRuleBuilder(hasFirstPart: boolean, hasLastPart: boolean, text: string): Rule {
+    return {
+        filter: (node: Node): boolean => {
+            return (node.parentNode as HTMLElement)?.id === 'turndown-root' && node.nodeName === 'TABLE' && (/\b(js|blob|diff)-./).test((node as HTMLTableElement).className);
+        },
+        replacement: (): string => {
```

Comment:
What is the `NEW_LINE_REPLACEMENT` keyword being used for? Could we maybe have a comment explaining it either here or

Reply:
Sure, I can add a comment, it is because the turndown conversation swallow new lines and sometimes I need to generate them in the final result.

Reply:
I added a comment in the place where the replacement of the NEW_LINE_REPLACEMENT is done.